README.md

```diff
@@ -25,51 +25,68 @@ Next up, the service provider must be registered:
 ```php
 'providers' => [
     ...
-    Spatie\FailedJobMonitor\FailedJobMonitorServiceProvider::class,
+    Spatie\FailedJobMonitor\ServiceProvider::class,
     ...
 ];
 ```
 
 Next, you must publish the config file:
 
 ```bash
-php artisan vendor:publish --provider="Spatie\FailedJobMonitor\FailedJobMonitorServiceProvider"
+php artisan vendor:publish --provider="Spatie\FailedJobMonitor\ServiceProvider"
 ```
 
-This is the contents of the configuration file. Most options are self-explanatory.
+This is the contents of the default configuration file. 
 
 ```php
 return [
+    '*' => [
+        'notifiable' => \App\User::class,
```

Comment:
I don't like the fact that de default notifiable has to be a user of the app.

 Replace this by a notifiable like the one in [laravel-backup](https://github.com/spatie/laravel-backup/blob/master/config/laravel-backup.php#L83)